src/Impact.php

```diff
@@ -801,6 +801,8 @@ public static function loadLibs()
     {
         echo Html::css('public/lib/cytoscape.css');
         echo Html::script("public/lib/cytoscape.js");
+        echo Html::script("public/lib/popper.min.js");
+        echo Html::script("public/lib/cytoscape-popper.js");
```

Comment:
These files should be imported by the `lib/bundles/cytoscape.js` file.

Reply:
Like this ?